### PR TITLE
Small refactor to the site

### PR DESCRIPTION
**Pull request by adrian-kong**
For #1819 
- Fixed the broken images (?)
- Created separate pages for huge chunks of information

### CODE REVIEW

**Comment by adrian-kong** [comment]
there wasn't much to add/change because oshi is already so simple to use by reading the javadocs but i just thought separating out the information would be better to read instead of a huge chunk on README that needs to scroll through for new users maybe

**Review by mprins — APPROVED** [review]
personally I'm not too keen on all those capitalised filenames, but others may think differently

**Comment by adrian-kong** [comment]
> personally I'm not too keen on all those capitalised filenames, but others may think differently

same i just followed the README.html format also maybe the markdown folder could also use some refactoring with html files and png files

**Comment by adrian-kong** [comment]
also oops it shows 2 commits on the logs i have no clue what I just did
did i just merge an already merged PR

**Comment by dbwiddis** [comment]
One is a "merge commit" which is actually empty. The other was the actual commit.  You should be using "squash and merge" which combines all the individual commits into the one PR merge commit and keeps the log history cleaner.

Note you only changed the README for the site, which would need to actually be republished (it happens as part of a release) so nothing has changed yet.  The main README on the github home page is the same.  Also typically I've been copy/pasting the main readme to the SITE readme, so maintaining two separate versions adds a lot of overhead to releases.

I still think we might want to re-organize the main page to match, as I'm really not sure how much traffic the github pages output actually sees. 

Regarding the all caps filenames, that has been a consistent format at least on the main site homepage for key documents needed by developers, e.g., README, UPGRADING, CHANGELOG.   If we're reorganizing content we can probably point to a single source (e.g., under the site tree) referenced from the main page.

Also the "projects using OSHI" from the main page README was not copied into the site version, but it would be nice to stick all those users/usages onto their own page somewhere.

Bottom line: thanks for the PR but I think there's a bit more to do :)
